src/v2/Apps/FairOrginizer/Components/FairOrganizerLatestArticles.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react"
+import styled from "styled-components"
+import { Box, Button, Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import { FairEditorialItemFragmentContainer as FairEditorialItem } from "v2/Apps/Fair/Components/FairEditorial/FairEditorialItem"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { createFragmentContainer, graphql, _FragmentRefs } from "react-relay"
+import { getArticlesColumns } from "../helpers/getArticlesColumns"
+import { FairOrganizerLatestArticles_fairOrganizer } from "v2/__generated__/FairOrganizerLatestArticles_fairOrganizer.graphql"
+import { extractNodes } from "v2/Utils/extractNodes"
+
+interface FairOrganizerLatestArticlesProps {
+  fairOrganizer: FairOrganizerLatestArticles_fairOrganizer
+}
+
+export const FairOrganizerLatestArticles: React.FC<FairOrganizerLatestArticlesProps> = ({
+  fairOrganizer,
+}) => {
+  const { articles, name } = fairOrganizer
+
+  const nodes = extractNodes(articles)
+  const [latestArticle, ...otherArticles] = nodes
+  const { leftColumn, rightColumn } = getArticlesColumns(otherArticles)
+
```

Comment:
**suggestion** (non-blocking): As an alternative to creating the articles columns helper, would something like this work as well?

```
const leftColumn = otherArticles.filter((v,i) => i % 2 === 0)
const reftColumn = otherArticles.filter((v,i) => i % 2 !== 0)
```

Reply:
That would cause double array iteration but as long as division into columns is not used elsewhere I think we can do without this helper and your variant fits as well 🙂

Reply:
Understood. I wasn't thinking the performance of my approach 😅 There's no need to make any changes on this comment.